src/js/bellows.js

```diff
@@ -52,6 +52,12 @@
             this._bindEvents();
         },
 
+        destroy: function() {
+            this.$bellows
+                .removeData(this.name)
+                .off(this.options.event);
```

Comment:
In shade (I believe it was shade) we unbound the specific handler. I would personally happy with doing both, we don't have the event namespaced yet and we should add that.